modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -56,7 +56,11 @@ You can estimate how much space the [path]``/var/spacewalk/`` directory requires
 
 |
 | [path]``/var/spacewalk``
-| Minimum 50{nbsp}GB per {suse} product and 360{nbsp}GB per Red Hat product
+| Minimum 100{nbsp}GB (the implemented check ensure it)
```

Comment:
Minimum storage required: 100 GB (this has been verified by our implemented check).